recipes/elasticsearch_set_index_settings.rb

```diff
@@ -0,0 +1,61 @@
+#
+# Cookbook:: elasticsearchwrapper
+# Recipe:: elasticsearch_set_replica
+#
+# Copyright:: 2018, BaritoLog.
+#
+#
+
+port = node['elasticsearch']['port']
+ipaddress = node['ipaddress']
+xpack_enabled = node['elasticsearch']['xpack_security_enabled']
+number_of_shards = node['elasticsearch']['index_number_of_shards']
+number_of_replicas = node['elasticsearch']['index_number_of_replicas']
+refresh_interval = node['elasticsearch']['index_refresh_interval']
+bootstrap_password = node['elasticsearch']['bootstrap_password']
```

Comment:
it is better move this in user side, user need to supplied value with encoded password already, and code only need to decode

Reply:
so it's better to not set a default password?

Reply:
yes, please remove default password, user may not need password.

Reply:
done, i removed the default bootstrap_password and change the recipe to decode base64 value from user